src/ddmd/semantic.d

```diff
@@ -38,12 +37,10 @@ extern(C++) void semantic(Dsymbol dsym, Scope* sc)
     dsym.accept(v);
 }
 
-// entrypoint for semantic ExpressionSemanticVisitor
 extern(C++) Expression semantic(Expression e, Scope* sc)
 {
-    scope v = new ExpressionSemanticVisitor(sc);
-    e.accept(v);
-    return v.result;
+    import ddmd.expressionsem;
```

Comment:
You could have used a selective import here.

Reply:
I'm going to delete the function shortly anyway.